docs/Diagnostics/HealthChecker/SerializedDataSigningCheck.md

```diff
@@ -12,6 +12,8 @@ The HealthChecker check validates that the feature is enabled on supported Excha
 
 ### Important
 
+ > :warning: **If you have an Exchange Server 2013 in your environment**: Turning on the signing of serialization payload feature might lead to several issues impacting management in your organization. We recommend not to turn on this feature for now. We will address this in the future update. Customers with Exchange Server 2016 / 2019 only can proceed with using the certificate signing of PowerShell serialization payload feature.
```

Comment:
This isn't going to be a warning section. 

Need to use the format followed in here: 

https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=warning#octicons